Cargo.toml

```diff
@@ -34,6 +34,7 @@ postgres-native-tls = "0.3"
 native-tls = "0.2"
 serde_path_to_error = "0.1.2"
 octocrab = "0.5"
+comrak = "0.8.2"
```

Comment:
Can we avoid using a different parser here? parser uses pulldown-cmark currently.

Reply:
From what I can see `pulldown-cmark` doesn't have an AST, and also doesn't have a way to generate markdown as output instead of HTML :(

Reply:
Sounds good. Seems fine then.